cmd/minishift/cmd/start_preflight.go

```diff
@@ -357,13 +357,22 @@ func checkHypervDriverSwitch() bool {
 
 // checkHypervDriverInstalled returns true if Hyper-V driver is installed
 func checkHypervDriverInstalled() bool {
+	// Check if Hyper-V's Virtual Machine Management Service is installed
```

Comment:
`// check` Let's start comment with small letter.

Reply:
No!

code comments start with lowercase, as it follows the casing. But textual comments should follow grammtic rules, so a sentence starts with a uppercase.

Reply:
sure